src/odemis/acq/align/test/align_test.py

```diff
@@ -0,0 +1,203 @@
+# -*- coding: utf-8 -*-
+
+"""
+This file is part of Odemis.
+
+Odemis is free software: you can redistribute it and/or modify it under the terms
+of the GNU General Public License version 2 as published by the Free Software
+Foundation.
+
+Odemis is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with
+Odemis. If not, see http://www.gnu.org/licenses/.
+"""
+from concurrent import futures
+from concurrent.futures._base import CancelledError
+import logging
+from odemis import model
+
+from numpy.core.multiarray import ndarray
+
+from odemis.acq.align import turnLightAndCheck
+from odemis.driver import simulated
+from odemis.dataio import tiff, hdf5
+from odemis.util import mock, img
+from odemis.util import TimeoutError
+import os
+import time
+import unittest
+import numpy
+import cv2
+
+
+logging.basicConfig(format="%(asctime)s  %(levelname)-7s %(module)-15s: %(message)s")
+logging.getLogger().setLevel(logging.DEBUG)
+
+
+TEST_IMAGE_PATH = os.path.dirname(__file__)
+
+class TestTurnLightAndCheck(unittest.TestCase):
+    """
+    Test turnLightAndCheck functions
+    """
+
+    def setUp(self):
+        self.light = simulated.Light("Calibration Light", "brightlight")
+
+        self.data = tiff.read_data(os.path.join(TEST_IMAGE_PATH, "brightlight-off-slit-spccd-simple.ome.tiff"))
+        self.img_light_off = img.ensure2DImage(self.data[0])
```

Comment:
I'd suggest to just load here both the "off" and "on" spccd images. Then, in all the cases which need them, you can use them.